src/main/java/graphql/schema/PropertyDataFetcherHelper.java

```diff
@@ -37,9 +37,6 @@ public static Object getPropertyValue(String propertyName, Object object, GraphQ
     }
 
     public static Object getPropertyValue(String propertyName, Object object, GraphQLType graphQLType, DataFetchingEnvironment environment) {
-        if (object == null) {
-            return null;
-        }
```

Comment:
This helper code is called by PropertyDataFetcher who already does a null check - I know its a simple IF statement but still